srcpkgs/lua54-mediator/template

```diff
@@ -0,0 +1,47 @@
+# Template file for 'lua54-mediator'
+pkgname=lua54-mediator
+version=1.1.2
+revision=1
+hostmakedepends="luarocks-lua54 lua51 lua52 lua53 lua54"
+depends="lua54"
+_desc="Mediator pattern implementation for pub-sub management"
+short_desc="${_desc} (5.4.x)"
+maintainer="Luca Matei Pintilie <luca@lucamatei.com>"
+license="MIT"
+homepage="https://olivinelabs.com/mediator_lua/"
+distfiles="https://github.com/Olivine-Labs/mediator_lua/archive/refs/tags/v${version}-0.tar.gz"
+checksum=faf5859fd2081be4e9e4fb8873a2dc65f7eff3fd93d6dd14da65f8e123fcff9b
+
+# FIXME: No license is provided by upstream. What do we put in the package?
```

Comment:
I think so you may copy text from https://opensource.org/license/mit

and paste in file and provide it in package